include/deployer.bash

```diff
@@ -199,6 +199,46 @@ cbd-update-snap() {
     debug $SELF_EXECUTABLE is updated
 }
 
+cbd-update-to-dev() {
+    declare desc="Updates CBD to the latest CB version on mow-dev"
+
+    if [[ -z "$DEV_URL" ]]; then
+        error "You did not supply a dev url in your profile! Update your profile to have a DEV_URL variable pointed to the desired URL."
+        _exit 1
+    fi
+
+    local latest_build=$(curl -s "${DEV_URL}/cloud/cb/info" | jq -r ".app.version")
+    cbd-update-to-build "$latest_build"
+}
+
+cbd-update-to-build() {
+    declare desc="Updates CBD to the specified build"
+    local build="$1"
+    local binver=$(bin-version)
+    local osarch=$(uname -sm | tr " " _ )
+    debug $desc
+    debug binver=$binver lastver=$lastver osarch=$osarch | gray
+
+    if [[ -z "$build" ]]; then
+        error "You did not supply a build! Update terminated"
+        _exit 1
+    fi
+
+    if [[ ${binver} != ${build} ]]; then
+        debug upgrade needed | yellow
+
+        local url=$(echo "https://public-repo-1.hortonworks.com/HDP/cloudbreak/cloudbreak-deployer_${build}_${osarch}.tgz")
```

Comment:
nit: why echo?

`local url="https://public-repo-1.hortonworks.com/HDP/cloudbreak/cloudbreak-deployer_${build}_${osarch}.tgz"`

Reply:
Good question, I don't see any reason for that I think I left it in by accident. Let me remove it and test again.